SPECS/coreutils/coreutils.spec

```diff
@@ -96,6 +98,9 @@ LANGUAGE=en_US.UTF-8 LC_ALL=en_US.UTF-8 make -k check
 %defattr(-,root,root)
 
 %changelog
+* Sat Feb 18 2023 Bala <balakumaran.kannan@microsoft.com> 8.30-12
+- Skip pwd-long.sh test which is failing in chroot
```

Comment:
Does this test only fail in the `chroot` and pass in 1.0 VM ?

Reply:
My bad. I tested it in Mariner-2.0 VM instead of 1.0. **This test is failing in 1.0 VM as well**

`pwd-long.sh:  does not contain old CWD
+ fail=255
+ Exit 255
+ set +e
+ exit 255
+ exit 255
+ remove_tmp_
+ __st=255
+ cleanup_
+ :
+ test '' = yes
+ cd /home/bala/coreutils-8.30
+ chmod -R u+rwx /home/bala/coreutils-8.30/gt-pwd-long.sh.VL63
+ rm -rf /home/bala/coreutils-8.30/gt-pwd-long.sh.VL63
+ exit 255
FAIL tests/misc/pwd-long.sh (exit status: 255)`